### PR TITLE
add FHist.jl and option to parse directly to `FHist.jl` objects

### PR DESCRIPTION
**Pull request by Moelf**

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -86,20 +86,36 @@ function JaggType(f, branch, leaf)
...
+function parseTH(th::Dict{Symbol, Any}; raw=true)
     xmin = th[:fXaxis_fXmin]
     xmax = th[:fXaxis_fXmax]
     xnbins = th[:fXaxis_fNbins]
     xbins = isempty(th[:fXaxis_fXbins]) ? range(xmin, xmax, length=xnbins+1) : th[:fXaxis_fXbins];
     counts = th[:fN]
+    nentries = th[:fEntries]
     sumw2 = th[:fSumw2]
-    if th[:fYaxis_fNbins] > 1
+    dimention = th[:fYaxis_fNbins]
```
Comment (Moelf) [inline src/utils.jl:117]:
```suggestion
    dimension = th[:fYaxis_fNbins]
```

File: src/utils.jl [hunk 2]
```diff
@@ -112,7 +128,13 @@ function parseTH(th::Dict{Symbol, Any})
         sumw2 = sumw2[2:end-1]
         edges = (xbins,)
     end
-    return counts, edges, sumw2
+    if raw
+        return counts, edges, sumw2, nentries
+    elseif dimention > 1
```
Comment (Moelf) [inline src/utils.jl:133]:
```suggestion
    elseif dimension > 1
```

**Comment by Moelf** [comment]
since there are a bunch of more information from `TH*` than what `StatsBsae.Histogram` can hold, I incline to include this so users don't have to copy paste a snippet every time.

**Comment by tamasgal** [comment]
I fully support that!

**Review by tamasgal — APPROVED** [review]

**Pull request merged**